test/unit/teacher_set_test.rb

```diff
@@ -116,6 +120,49 @@ class TeacherSetTest < ActiveSupport::TestCase
     end
   end
 
+  # Test current-user teacher-set holds quantity count
+  describe "test teacher-set holds count" do
+    # Test-1
```

Comment:
These comments don't really add anything — I think it's better if you take them out. Thanks!